db/migrate/20210810105233_remove_reservation_statuses_table.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveReservationStatusesTable < ActiveRecord::Migration[6.0]
+  def change
+    drop_table :reservation_statuses
```

Comment:
How about just removing migration for creating reservation tickets and re-create the database?